### ISSUE TITLE
createBottomTabNavigator with initialRouteName

### ISSUE THREAD

**Issue by kapi-no** [react-navigation/tabs#162]
I just migrated to the newest release of tabs because of Typescript support. Previously, I used old version of react-navigation 3.11.1 where tab navigators were part of the module. However, I found out that the new version does not support the style of configuration in the ```createBottomTabNavigator()```

### Current behaviour

Second parameter of ```createBottomTabNavigator()``` for ```NavigationBottomTabOptions``` does not accept ```intialRouteName``` field.

Argument of type '{ initialRouteName: string; }' is not assignable to parameter of type 'NavigationBottomTabOptions'.

### Expected behaviour

It should be possible to set ```initialRouteName``` when creating the ```BottomTabNavigator```.

### Code sample

```
const eventTab = createBottomTabNavigator(
{
    EventDetail: {
        screen: EventDetailScreen,
    },
},
{
    initialRouteName: 'EventDetail',
});
```

### Your Environment

| software              | version
| --------------------- | -------
| android        | 8.0+
| react-native          | 0.60.5
| react-navigation-tabs | 2.4.1

**Comment by tyanbiao** [comment]
I had the same problem.

**Comment by pellul** [comment]
Downgrading `react-navigation-tabs` to 2.4.0 fixed this problem for me.

**Comment by jakubukrop** [comment]
There could be a problem with typescript definition - I had the same problem with `createMaterialTopTabNavigator` too. Other props such as `tabBarPosition` were also not accepted.

**Comment by 15matias15** [comment]
Same issue here

**Comment by satya164** [comment]
Closing in favor of #163

**Comment by kapi-no** [comment]
This is solved in the 2.5.3 version. Many thanks :)
